### PR TITLE
[ci] build arm64 docker images

### PR DESCRIPTION
**Pull request by can-anyscale**
Build arm64 docker images in civ2

Test:
- CI

### CODE REVIEW

File: .buildkite/_forge.aarch64.rayci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          - "3.11"
+        cuda:
+          - "11.5.2"
+          - "11.6.2"
+          - "11.7.1"
+          - "11.8.0"
+          - "12.1.1"
```
Comment (aslonnie) [inline .buildkite/_forge.aarch64.rayci.yml:30]: I do not think we build any cuda versions for arm.

I am surprised that the bases even work here.

**Review by aslonnie — COMMENTED** [review]
are we removing the build jobs on civ1?

**Comment by can-anyscale** [comment]
w00t yes, need to remove civ1 build jobs too. look like they build the cuda versions as well: https://github.com/ray-project/ray/blob/master/.buildkite/pipeline.arm64.yml#L44

**Comment by aslonnie** [comment]
> w00t yes, need to remove civ1 build jobs too. look like they build the cuda versions as well: https://github.com/ray-project/ray/blob/master/.buildkite/pipeline.arm64.yml#L44

hah, I have always thought that gpu on arm is not really a thing, it seems that ec2 actually has g5g instances, and I was wrong.

**Comment by aslonnie** [comment]
and we build even more images that I thought..

**Review by aslonnie — APPROVED** [review]

**Pull request merged**